tensorflow-tools/src/main/java/org/tensorflow/tools/Shape.java

```diff
@@ -19,16 +19,29 @@
 
 import java.util.Arrays;
 
+/**
+ * The shape of a Tensor or {@link org.tensorflow.tools.ndarray.NdArray}. Defines the size along
+ * its axis. May contain an unknown size for one of the axis or may be totally unknown, in which
+ * case not even the number of axis is known.
+ */
 public final class Shape {
 
+  /**
+   * Size reported for shapes for which the total size cannot be computed, either because their
+   * number of dimensions is unknown or one or more of their dimensions have an unknown size.
+   */
   public static long UNKNOWN_SIZE = -1L;
 
-  /** Create a Shape representing an unknown number of dimensions. */
+  /** Create a Shape representing an unknown number of dimensions.
+   * @return a Shape for which {@link Shape#isUnknown()} is true.
+   */
```

Comment:
Here, the [guide](https://google.github.io/styleguide/javaguide.html#s7.1.1-javadoc-multi-line) suggest to enter a newline after `/**` (same thing, should be applied to all javadoc blocks added)

Reply:
Done!